code/utils.py

```diff
@@ -156,7 +156,7 @@ def save_user_embeddings(email_ids, embeddings):
     the method saves the email embeddings passed to it as a pickle file
     :return: saves as pickle file
     """
-    filepath = '../resources/embeddings.pkl'
+    filepath = '../resources/' + 'embeddings_' + constants.RUN_ID + '.pkl'
```

Comment:
Now we will have a lot of embeddings corresponding to each experiment
Should we save them in the log folder?